### PR TITLE
Add serviceable attribute to projects.

### PR DESCRIPTION
**Pull request by NTaylorMullen**

### CODE REVIEW

File: src/Microsoft.Framework.DependencyInjection.Autofac/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -0,0 +1,6 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Reflection;
+
+[assembly: AssemblyMetadata("Serviceable", "True")]
```
Comment (Eilon) [inline src/Microsoft.Framework.DependencyInjection.Autofac/Properties/AssemblyInfo.cs]: Sample project, so skip.

File: src/Microsoft.Framework.DependencyInjection.Ninject/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -0,0 +1,6 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Reflection;
+
+[assembly: AssemblyMetadata("Serviceable", "True")]
```
Comment (Eilon) [inline src/Microsoft.Framework.DependencyInjection.Ninject/Properties/AssemblyInfo.cs]: Sample project, so skip.

File: src/Microsoft.Framework.DependencyInjection.StructureMap/Properties/AssemblyInfo.cs [hunk 3]
```diff
@@ -0,0 +1,6 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Reflection;
+
+[assembly: AssemblyMetadata("Serviceable", "True")]
```
Comment (Eilon) [inline src/Microsoft.Framework.DependencyInjection.StructureMap/Properties/AssemblyInfo.cs]: Sample project, so skip.

File: src/Microsoft.Framework.DependencyInjection.Unity/Properties/AssemblyInfo.cs [hunk 4]
```diff
@@ -0,0 +1,6 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Reflection;
+
+[assembly: AssemblyMetadata("Serviceable", "True")]
```
Comment (Eilon) [inline src/Microsoft.Framework.DependencyInjection.Unity/Properties/AssemblyInfo.cs]: Sample project, so skip.

File: src/Microsoft.Framework.DependencyInjection.Windsor/Properties/AssemblyInfo.cs [hunk 5]
```diff
@@ -0,0 +1,6 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Reflection;
+
+[assembly: AssemblyMetadata("Serviceable", "True")]
```
Comment (Eilon) [inline src/Microsoft.Framework.DependencyInjection.Windsor/Properties/AssemblyInfo.cs]: Sample project, so skip.

**Comment by NTaylorMullen** [comment]
We should make a samples folder here. Kinda lame that it's all intermingled in the src folder.

**Comment by Eilon** [comment]
In dev some of these are gone already anyway. Eventually they will all be gone. But yeah it's a bit of a mess.

**Comment by NTaylorMullen** [comment]
Updated.

**Comment by Eilon** [comment]
:shipit:

**Pull request merged**